tests/cli-self-upd.rs

```diff
@@ -343,6 +343,20 @@ fn install_does_not_add_path_to_bash_profile_that_doesnt_exist() {
     });
 }
 
+#[test]
+#[cfg(unix)]
+fn install_errors_when_rc_file_cannot_be_updated() {
+    setup(&|config| {
+        let rc = config.homedir.join(".bash_profile");
+        fs::File::create(&rc).unwrap();
```

Comment:
I felt like there was a lot of `unwrap`ping here, but I wasn't sure what the best way to handle that might be, or if others might even see it as a problem.

Reply:
Its a test; meh :)